### ISSUE TITLE
When Updating Ubuntu VMs the output isn't parsed nicely

### ISSUE THREAD

**Issue by Botched1** [BassT23/Proxmox#49]
When updating Ubuntu Server with no desktop experience, the update output ism formatted nicely, example:
```
[Info] Starting LXC 101 
[Info] Waiting for LXC 101 to start 
[Info] Updating LXC 101 : ustest

--- APT UPDATE ---
Hit:1 http://archive.ubuntu.com/ubuntu jammy InRelease
Get:2 http://archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy-security InRelease [110 kB]
Get:4 http://archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [942 kB]
Get:5 http://archive.ubuntu.com/ubuntu jammy-updates/main Translation-en [204 kB]
```

When updating an Ubuntu Desktop experience system, though, the output is not formatted. Mainly it looks like new line escapes are not parsed? Example:
```
[Info] Starting VM 102 
[Info] Waiting for VM 102 to start
This will take some time, ... 30 secounds is set!
[Info] Updating VM 102 : UD-test

--- APT UPDATE ---
"Hit:1 http://us.archive.ubuntu.com/ubuntu jammy InRelease\nGet:2 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease [119 kB]\nGet:3 http://security.ubuntu.com/ubuntu jammy-security InRelease [110 kB]\nGet:4 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease [107 kB]\nGet:5 http://us.archive.ubuntu.com/ubuntu jammy-updates/main amd64 Packages [942 kB]\nGet:6 http://security.ubuntu.com/ubuntu jammy-security/main i386 Packages [269 kB]\nGet:7 http://us.archive.ubuntu.com/ubuntu jammy-updates/main i386 Packages [455 kB]\nGet:8 http://security.ubuntu.com/ubuntu jammy-security/main amd64 Packages [686 kB]\nGet:9 http://us.archiv
```

**Comment by BassT23** [comment]
The problem is the used "qemu" method.
Its good to use, because no additional infos are need for a VM.

The other output will work, if I change to a ssh connection, but then I need the connection infos.
Is planned for later release, but will be hard to implement, because will need an additional method with encrypted connection infos, changed by user.

**Comment by BassT23** [comment]
if you want,
look here: <https://github.com/BassT23/Proxmox/blob/development/ssh.md>
and test development branch

**Comment by BassT23** [comment]
so, now its better.
please test latest beta with
`bash <(curl -s https://raw.githubusercontent.com/BassT23/Proxmox/development/install.sh) update`

Infos for ssh connection can be found here:
`https://github.com/BassT23/Proxmox/blob/development/ssh.md`

**Comment by BassT23** [comment]
thats development :D the cluster dont work now.
Sry, need some fixing. The Problem is the new implemented cleaning, ...

**Comment by BassT23** [comment]
now it should work. I tested myself and all work well.
Please test and give feedback ;)
`bash <(curl -s https://raw.githubusercontent.com/BassT23/Proxmox/development/install.sh) update`

**Comment by BassT23** [comment]
implemented in v3.8
